app/views/content_items/short_text/_just_govspeak.html.erb

```diff
@@ -0,0 +1,13 @@
+<% content_for :simple_header, true %>
```

Comment:
Technically this isn't _just govspeak_, it's a title, body and related links.
Maybe body_with_related_links, or something similar? This would be consistent with the names of other partials.